PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-detail/pokemon-detail.component.ts

```diff
@@ -46,14 +49,15 @@ export class PokemonDetailComponent implements OnInit {
             this.pokemonService.getByNameFull(this.model.pokemonName).subscribe((result) => {
                 this.model.pokemon = result as IPokemonVarietyModel;
 
-                this.titleService.setTitle(`${this.model.pokemon.name} - ${WEBSITE_NAME}`);
+                    this.titleService.setTitle(`${this.model.pokemon.name} - ${WEBSITE_NAME}`);
 
-                this.model.learnableMoves = this.model.pokemon.learnableMoves;
-
-                this.sortMoveLearnMethods();
-                this.sortForms();
-                this.applyInitialSorting();
-            });
+                    this.model.learnableMoves = this.model.pokemon.learnableMoves;
+                    
+                    this.hideEventExclusiveSpawns();
```

Comment:
As answer to your question about the if-statement here: No, if the default behaviour changes, you would just have to call "showEventExclusiveSpawns()" here instead of "hide...". The initial value that is set in the model is irrelevant, as both methods (hide/show) update the boolean value anyway.